src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -230,21 +230,27 @@ private void UpdateIssues()
 
         private void UpdateCaption()
```

Comment:
I was able to reproduce it consistently.

`UpdateCaption` is called from 3 places:
1. CTOR (runs on the UI thread - no problem)
2. `ActiveDocumentTracker_OnDocumentFocused` (runs on the UI thread - no problems)
3. `Store_IssuesChanged` -- might run on background thread, which then causes this problem.

The third scenario gets broken when the window already exists by the time the synchronizer runs (reproducible by opening the tool window quickly while the binding still loads). 

I was considering placing the call to `RunOnUIThread` inside `Store_IssuesChanged`, but then we are at risk for it to be broken again in the future, if we call it from a 4th place that runs on the background thread.
I also thought about wrapping just `GetFilteredIssuesCount` but this seemed easier.
Let me know what you think.